llvm/docs/CommandGuide/lit.rst

```diff
@@ -151,6 +151,11 @@ EXECUTION OPTIONS
  feature that can be used to conditionally disable (or expect failure in)
  certain tests.
 
+.. option:: --skip-test-recording
+
+ Disable tracking the wall time individual tests take to execute. This allows
+ external build systems to orchestrate the scheduled tests.
```

Comment:
I feel like the option name should have "time" in the title somewhere, e.g. `--skip-test-time-recording`.